proxy.py

```diff
@@ -11,61 +11,96 @@ def __init__(self):
         super().__init__()
         self.logger = get_nio_logger("ConfigProxy")
 
-    def get_version(self, url_prefix, config_id, apikey):
-        """
-        Retrieves the latest instance configuration version id
+    def get_instance_config_ids(self, url_prefix, instance_id, apikey):
+        """ Gets the conf id and conf version id the instance should be running
+
+        Args:
+            url_prefix (str): url prefix to product api
+            instance_id (str): instance id
+            apikey (str): api key
 
         Returns: an object with format
             {
+                "instance_configuration_id": "uuid..",
                 "instance_configuration_version_id": "uuid..",
                 "version_num": "v1.0.0",
                 "created_at": "2018-03-14 12:12:12"
```

Comment:
`version_num` and `created_at` are no longer returned by this enpoint. Just the two UUIDs